Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,364 @@
+
+## Conclusion
+
+A wide range of open-source container registry projects (Quay, Harbor, Dragonfly,
+Keppel, Portus, Kraken, etc.) has been carefully evaluated based on the two main
+factors: the open-source health and range of supported features.
+
+The open-source software health is crucial and container registry implementation should
+pass it to be SCS-compliant. The OSS health check evaluates several important metrics
+of an open source software like whether the code/community/development/design is
+fully open or whether the project's maturity, security, and activity are on the desired
+level. This check also evaluates the lock-in risk due to possible single points of
+failure or internal project conflicts and several other aspects.
+Overall, three projects passed the OSS health checks:
+- [Harbor][harbor]
+- [Project Quay][projectquay]
+- [Dragonfly][dragonfly]
+
+The above projects were then evaluated from the "supported features" perspective.
+The [Required and desirable features check](#required-and-desirable-features-check)
+did an overview of the feature set of open-source container registry implementations
+and map SCS requirements (and nice-to-haves) against it. The list of required features
+is quite long and contains features that are primarily focused on security
+(authentication, vulnerability scanning, content trust, and validation, etc.),
+scalability (HA mode, registry replication, p2p integration, etc.) and visibility
+(monitoring), see the full list [here](#required-and-desirable-features-check).
+These requirements should ensure that the selected container registry implementation
+could be offered by the CSPs as a secure and enterprise-ready solution.
+
+Let's discuss and compare the projects Dragonfly, Quay, and Harbor in the following
+section.
+
+[Dragonfly][dragonfly] is a healthy open-source project with a growing community
+and CNCF incubating maturity level. It is considered stable, and it is used by many
+companies in their production environments. We currently see that it is not as
+feature-rich as Harbor or Quay, hence it is not considered the best choice here.
+It seems, that its main aim (currently) is to offer (an efficient, stable, and secure)
+container distribution solution based on p2p technology. This improves download
+efficiency and saves bandwidth across CSPs. It also offers integration possibilities
+that allow one to use it as a p2p distribution network via a "preheat" API. This
+integration was implemented in the Harbor project via Dragonfly "preheat" adapter, and
+both parties may benefit from the integration. The Harbor profits from Dragonfly's p2p
+distribution capabilities and on the other hand the Dragonfly project profits from
+Harbor's feature-rich container registry "frontend".
+
+[Quay][projectquay] is the Red Hat open-source project. Its OSS health is on a good level,
```

Comment:
I wouldn't say that it is "the" Red-Hat open-source project as I'd consider that to be RHEL, so it might be better to simply say `[Quay][projectquay] is an open-source project maintained by Red Hat` or similar (depending on how much "Red Hat"'s involvement should be emphasised or is factually correct).